Source/MonoGame.Extended.Tests/AngleTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using NUnit.Framework;
+
+namespace MonoGame.Extended.Tests
+{
+    [TestFixture]
+    public class AngleTest
+    {
+        private const float DELTA = 0.0000001f;
```

Comment:
Nowhere in the [C# naming conventions](https://msdn.microsoft.com/en-us/library/x2dbyw72(v=vs.71).aspx) does it say to use ALL_CAPS. The [internet agrees](http://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants) with this. So these constants should use standard field or property naming `Delta` or `_delta`.